embedded-btfhub.go

```diff
@@ -0,0 +1,16 @@
+//go:build ebpf && btfhub
+// +build ebpf,btfhub
+
+package tracee
+
+import (
+	"embed"
+)
+
+var BTFHubEmbedded bool = true
+
+//go:embed "dist/tracee.bpf/*"
+//go:embed "dist/tracee.bpf.core.o"
+//go:embed "dist/btfhub/*"
```

Comment:
So, this file is the same as embedded-ebpf.go except this line, right? Maybe we can just add this line to the other file, embedding an empty dir (which we can create in the makefile) and avoid adding this new file?

Reply:
Yes, that was my initial idea in the other PR that was closed. There was a `touch ./dist/btfhub/place-holder`, maybe you recall, as empty dirs can't be embedded. I do agree that either having:

1. ./dist/btfhub/place-holder
2. ./dist/btfhub/{ubuntu,centos,fedora}

is better than this embedded-btfhub.go logic. Let me go back to initial idea then.